docs/environment.yml

```diff
@@ -2,6 +2,7 @@ name: readthedocs
 channels:
   - conda-forge
 dependencies:
+  - python<3.13
```

Comment:
Some bit of Sphinx/RtD code uses the Python standard-library `imghdr` module, which was deprecated by [this PEP](https://peps.python.org/pep-0594/#imghdr) and has been removed in Python 3.13, which RtD is using for its builds by default. It's possible that updating/relaxing the pins on `sphinx_rtd_theme` and `sphinxcontrib-bibtex`, below, might also fix this, as newer versions of those packages might deal with the `imghdr` issue. But I think this is the simplest immediate fix for the issue. We will have a number of other tasks to attend to (probably including relaxing some dependency-package pins in `meta.yaml`) when we extend `uwtools` support to include Python 3.13, so I propose that we deal with them all at once, maybe in our next `uwtools` release.